flang/test/Parser/OpenMP/scan.f90

```diff
@@ -0,0 +1,40 @@
+! RUN: %flang_fc1 -fdebug-unparse -fopenmp %s | FileCheck --ignore-case %s
+! RUN: %flang_fc1 -fdebug-dump-parse-tree -fopenmp %s | FileCheck --check-prefix="PARSE-TREE" %s
+
+! Check for parsing scan directive
+subroutine test_scan_inclusive()
```

Comment:
Nit: It's testing both types of scan, so I think it's best to avoid a misleading subroutine name.
```suggestion
subroutine test_scan()
```

Reply:
Updated.